SettingsList.py

```diff
@@ -4547,6 +4547,25 @@ class SettingInfos:
         default        = False,
     )
 
+    credits_music = Checkbutton(
```

Comment:
I also didn't see this mentioned in the PR title or description

Reply:
oh yea that's my bad its just the little button to enable the credits music being randomized so its optional.